JMS/src/test/java/com/ibm/mq/samples/jms/SampleEnvSetterTest.java

```diff
@@ -17,139 +17,67 @@
 package com.ibm.mq.samples.jms;
 
 import static org.junit.jupiter.api.Assertions.*;
-import java.lang.reflect.Field;
-
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.BeforeAll;
-import org.json.JSONArray;
-import org.json.JSONObject;
-import java.util.ArrayList;
-import java.util.List;
 
 public class SampleEnvSetterTest {
```

Comment:
These are private strings in `SampleEnvSetter`. 

```
    private static final String ENV_FILE = "EnvFile";
    private static final String DEFAULT_ENV_FILE = "../env.json";    
    private static final String DEFAULT_Z_ENV_FILE ="../env-zbindings.json";
``` 
In this case, there is an argument to make them public. So the code in the test makes use of them.